web/templates/layout/app.html.eex

```diff
@@ -51,7 +51,7 @@
           <div class="dtc-ns w-50-l w-40-m v-mid">
             <%= transform_html @banner, @conn, class: "underline lm-white dib", id: "feedback-link" %>
           </div>
-          <div class="dtc-ns dn tr v-mid">
+          <div class="dtc-ns dn tr v-mid banner_buttons">
```

Comment:
From these classes `dtc-ns dn tr v-mid`

We're making them disappear with the default being `dn` (`display: none`) and `dtc-ns` (`display: table-cell` with media queries for `not small` screens)